utils/handlers/graphql.go

```diff
@@ -0,0 +1,36 @@
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/spaceuptech/space-cloud/model"
+	"github.com/spaceuptech/space-cloud/utils/graphql"
+)
+
+// HandleCrudCreate creates the create operation endpoint
+func HandleGraphQLRequest(graphql *graphql.Module) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+
+		// Create a context of execution
+		_, cancel := context.WithTimeout(r.Context(), 5*time.Second)
+		defer cancel()
+
+		// Get the path parameters
+		_ = getRequestMetaData(r)
```

Comment:
Use the project returned by this method to match with the project stored in the GraphQL module. Also, pass the token in the ExecGraphQLQuery method. We need to pass the token to wherever the auth module is used.